### PR TITLE
More various fixes

### PR DESCRIPTION
**Pull request by husobee**
### Summary

More issue resolution.  Resolves:
fixes: #494 
fixes: #496 

### Type of change ( select one )

- [ ] Product feature
- [x] Bug fix
- [ ] Performance improvement
- [ ] Refactor
- [ ] Other

### Tested Environments

- [ ] Development
- [ ] Staging
- [ ] Production

### Before submitting this PR:

- [ ] Does your code build cleanly without any errors or warnings?
- [ ] Have you used auto closing keywords?
- [ ] Have you added tests for new functionality?
- [ ] Have validated query efficiency for new database queries?
- [ ] Have documented new functionality in README or in comments?
- [ ] Have you squashed all intermediate commits?
- [ ] Is there a clear title that explains what the PR does?
- [ ] Have you used intuitive function, variable and other naming?
- [ ] Have you requested security / privacy review if needed
- [ ] Have you performed a self review of this PR?

### Manual Test Plan:

N/A

### CODE REVIEW

File: utils/wallet/provider/uphold/uphold.go [hunk 1]
```diff
@@ -115,8 +115,9 @@ func init() {
 		Timeout: time.Second * 60,
 		Transport: middleware.InstrumentRoundTripper(
 			&http.Transport{
-				Proxy:   proxy,
-				DialTLS: pindialer.MakeDialer(upholdCertFingerprint),
+				Proxy:          proxy,
+				DialTLSContext: pindialer.MakeContextDialer(upholdCertFingerprint),
+				DialTLS:        pindialer.MakeDialer(upholdCertFingerprint),
```
Comment (maikelmclauflin) [inline utils/wallet/provider/uphold/uphold.go:120]: don't think we need to set both here: 
```
If both are set, DialTLSContext takes priority.
```

**Review by maikelmclauflin — CHANGES_REQUESTED** [review]
overall looks good

File: utils/wallet/provider/uphold/uphold.go (hunk 1 above)
Comment (husobee) [reply utils/wallet/provider/uphold/uphold.go]: https://github.com/brave-intl/bat-go/pull/497/commits/4663efa6ccd9af5d3408d5106b480427e700df68 <- fixed here

**Review by evq — DISMISSED** [review]
Looking good!

**Review by maikelmclauflin — APPROVED** [review]

**Pull request merged**